Core/Src/torque.c

```diff
@@ -26,11 +30,62 @@
 #define MIN_COMMAND_FREQ  60					  /* Hz */
 #define MAX_COMMAND_DELAY 1000 / MIN_COMMAND_FREQ /* ms */
 
+static uint16_t torque_accumulator[ACCUMULATOR_SIZE];
```

Comment:
I would recommend making this a static variable _inside_ the function for calcs. This limits the namespace of the variable to only inside that function AND it exists for the lifetime of the program